guides/workspace-settings/labels.md

```diff
@@ -1,14 +1,25 @@
 # Labels
-Labels are great for your team's administration to organize certain people. 
+Labels are great for your team's administration to organize certain people. You can use them to label staff from their timezone, perks, or status.
 
-Some ideas may be:
-- Timezone
-- Partnerships/Development/Support/Events/Moderation Team
-- Perks
-- Up for Promotion/Demotion
+:::caution PERMISSION REQUIRED
+To proceed, please note that you need the following permissions: **Workspace Admin**
+:::
 
-![labels](/img/labels.PNG)
+## Creating Labels
+To create a label, go to Settings > Labels and click "Add Label."
+![](https://cdn.discordapp.com/attachments/1015246364692717609/1036967516381126726/unknown.png)
 
-Create your labels in your settings -> Go to the Staff tab -> Go on any view -> Open a staff profile -> On the details tab, click "+ Add Label".
+Once created, you can change it's color and name. 
+![](https://cdn.discordapp.com/attachments/1015246364692717609/1036968286593749063/unknown.png)
```

Comment:
Do not hotlink.

Reply:
Done. I've changed it to Hyra Drive link.

```
Once created, you can change it's color and name. 
![](https://drive.hyra.io/apps/files_sharing/publicpreview/7RFyp3DKrDD8cYJ?x=1879&y=616&a=true&file=labels-2.png&scalingup=0)
```